doc/api/hdr.md

```diff
@@ -0,0 +1,89 @@
+# HDR support ##################################################################
+
+## Overview ####################################################################
+
+HDR (High Dynamic Range) is a video technology that improves the way light is
+represented by permitting to render brighter highlights, darker shadows and
+more details beetwen both ends. Sometimes, it allows to reproduce richer colors
+than with the standard dynamic range.
+
+## API #########################################################################
+
+Behind this principle, several formats exists (HDR10, HLG, Dolby Vision) and
+implements specific media and stream encoding and packaging technologies. When
+using streaming technologies, both streaming manifest and codecs strings can
+provide information about the technical HDR characteristics of the content.
+
+These information are parsed and exposed through the ``hdrInfo`` that can be
+found in the periods/adaptation/representation path of the RxPlayer manifest
+object. Also, the ``getAvailableVideoTracks`` and ``getVideoTrack`` functions
+and the ``videoTrackChange`` event carries the ``hdrInfo``.
+
+### hdrInfo ####################################################################
+
+- colorDepth: ``number|undefined`` : It is the bit depth used for encoding the
+  color for a pixel. It may vary from 10 to 16. The more bits are used for
```

Comment:
Can it ever ever ever go lower than 10 or higher than 16?

I'm asking because we risk breaking the API if we one day encounter the case

Reply:
Nice for making it `undefined` here (though it is not for now in the API)

Reply:
I've just removed it, as it could take more values in the future.